ui/client/containers/plainStyleLink.tsx

```diff
@@ -0,0 +1,15 @@
+import {css} from "emotion"
+import React from "react"
+import {Link, LinkProps} from "react-router-dom"
+
+export function PlainStyleLink(props: LinkProps) {
```

Comment:
I think.. PlainStyleLink and ProcessLink is the same? Can we do one more generic component?